### PR TITLE
fix: fix upLoadList style

### PR DESCRIPTION
**Pull request by Veveue**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

#16045  #11183
### 💡 Solution

https://cdn.nlark.com/yuque/0/2019/png/171650/1558846709699-478609e6-c4f4-478b-ae2e-babc5b042a42.png

https://cdn.nlark.com/yuque/0/2019/png/171650/1558850660399-e0178c44-c0f6-4d74-93ab-019e1da7daed.png

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by ycjcl868** [comment]
image

it will flash when wrap into a new line.

File: components/upload/demo/avatar.md [hunk 1]
```diff
@@ -68,17 +68,19 @@ class Avatar extends React.Component {
...
-        listType="picture-card"
-        className="avatar-uploader"
-        showUploadList={false}
-        action="https://www.mocky.io/v2/5cc8019d300000980a055e76"
-        beforeUpload={beforeUpload}
-        onChange={this.handleChange}
-      >
-        {imageUrl ? <img src={imageUrl} alt="avatar" /> : uploadButton}
-      </Upload>
+      <div className="clearfix">
```
Comment (afc163) [inline components/upload/demo/avatar.md]: 最好不要依赖外部样式。

Comment (afc163) [reply components/upload/demo/avatar.md]: 不要依赖 `clearfix` 类名。

**Comment by afc163** [comment]
Merged in https://github.com/ant-design/ant-design/commit/913d19113fdc140ac9053abc7de4b0512b5cc4c1

**Pull request closed without merging**